app/models/form526_submission.rb

```diff
@@ -47,17 +47,18 @@ class Form526Submission < ApplicationRecord
   SUBMIT_FORM_526_JOB_CLASSES = %w[SubmitForm526AllClaim SubmitForm526].freeze
 
   def start
-    if single_issue_hypertension_claim? && Flipper.enabled?(:disability_hypertension_compensation_fast_track)
+    rrd_processor_class = rrd_process_selector.processor_class
+    if rrd_processor_class
       workflow_batch = Sidekiq::Batch.new
       workflow_batch.on(
         :success,
         'Form526Submission#start_evss_submission',
         'submission_id' => id
       )
-      jids = workflow_batch.jobs do
-        RapidReadyForDecision::DisabilityCompensationJob.perform_async(id, full_name)
```

Comment:
Instead of passing in the result of calling `full_name`, within the job's `perform` method, call `full_name` on the `Form526Submission` record identified by `id` . The user's full name is not expected to change between calling `full_name` here versus calling it within the job.